layers/+lang/flow-type/README.org

```diff
@@ -0,0 +1,38 @@
+#+TITLE: flow-type layer
+
+* Description
+This layer adds some [[https://flowtype.org/][flow]] related functionality to Emacs:
+ - Show the flow-type of the object under the cursor using Eldoc
+ - Add flow checking to flycheck (based on the flycheck-flow package)
+ - company-mode completion using company-flow
+ - "compilation" using =flow status= and =flow check=
+ - auto staring =flow server= in a project specific comint buffer
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =flow-type= to the existing =dotspacemacs-configuration-layers= list in this
+file. Please ensure you already have the flow binary installed and in your PATH.
+
+To prevent the flow server from auto-starting, set =flow-type-no-auto-start= to =t=.
+To start the server in a comint buffer on entering a =js2-mode= buffer, set it to ='process=.
+
+#+BEGIN_SRC emacs-lisp
+(flow-type :variables flow-type-no-auto-start 'process)
```

Comment:
Urgh, possibly.   I have two counter arguments:


1) `no-auto-start` matches the `flow` option  - from `flow --help`: 

```
  --no-auto-start  If the server is not running, do not start it; just exit
```

2) In its current form, the two possible truthy values for `flow-type-no-auto-start` ( `'process` & `t`)  are siblings - they both add `--no-auto-start` to the flow command.  With your suggestion, it'd be `'process` & `nil` that add `--no-auto-start` - one truthy and one falsey value, which feels a little weird.


They're not strong arguments though - if you still think it'd be better the other way round, I'll change it.